v0/sources/canny/transform.js

```diff
@@ -0,0 +1,82 @@
+const path = require("path");
+const fs = require("fs");
+const Message = require("../message");
+const sha256 = require("sha256");
+
+// import mapping json using JSON.parse to preserve object key order
+const mapping = JSON.parse(
+  fs.readFileSync(path.resolve(__dirname, "./mapping.json"), "utf-8")
+);
+
+function process(event) {
+  const message = new Message(`Canny`);
+
+  // event type is always track
+  const eventType = "track";
+
+  message.setEventType(eventType);
+
+  message.setPropertiesV2(event, mapping);
+
+  message.context.integration.version = "1.0.0";
+  try {
+    switch (event.type) {
+      case "vote.created":
+      case "vote.deleted":
```

Comment:
For these we can use separate mapping things will be cleaner in terms of code

Reply:
Added